### DISCUSSION TITLE
Sensor Thermocouple SHT30 not working despite Wiki says otherwise

### DISCUSSION THREAD

**Discussion by rubenkr** [arendst/Tasmota#14956] [Q&A] [answered]
### PROBLEM DESCRIPTION
Attached a SHT30 sensor inside a thermocouple for high humidity to a ESP8266 board. I am using the latest Tasmota 11.0.0.2 by Theo Arends. Followed Wiki Article [https://www.wemos.cc/en/latest/d1_mini_shield/sht30.html](url) for wiring as follows:

# Wiring SHT30
red = VCC 3,3V
yellow = SGK
blue = DATA
green = GND

# Wiring ESP82 -> SHT30
- SDA Pin of SHT30 to D2 
- SCL Pin of SHT30 to D1

# Tasmota Generic 
D2 GPIO4 -- I2C SCA
D1 GPIO5 -- I2C SCL

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____
- [x] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
16:29:11.599 CMD:   STATUS 0
16:29:11.604 RSL: STATUS = {"Status":{"Module":18,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_09BB66","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0}}
16:29:11.630 RSL: STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":" ","RestartReason":"Software/System restart","Uptime":"0T00:05:33","StartupUTC":"2022-02-23T15:23:38","Sleep":50,"CfgHolder":4617,"BootCount":6,"BCResetTime":"2022-02-23T16:12:23","SaveCount":14,"SaveAddress":"F6000"}}
16:29:11.659 RSL: STATUS2 = {"StatusFWR":{"Version":"11.0.0.2(allsensors)","BuildDateTime":"2022-02-23T12:19:48","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"310/699"}}
16:29:11.678 RSL: STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Vespa-O",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A190000000000","00000080","00006000","00004000"]}}
16:29:11.715 RSL: STATUS4 = {"StatusMEM":{"ProgramSize":718,"Free":284,"Heap":18,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"164020","FlashFrequency":40,"FlashMode":3,"Features":["00000809","8F908787","04146001","FFFFB7CF","8F7417FF","303E0AD3","1149785A","26204FFF","0008F0F3"],"Drivers":"1,2,3,4,7,8,9,10,12,14,15,20,21,24,28,29,34,43,47,50,56,60","Sensors":"1,2,3,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,24,26,27,28,29,30,31,32,33,38,40,41,42,44,45,46,50,51,54,55,56,57,58,59,63,64,65,66,67,70,71,72,73,74,76,78,79,81,82,85,86,88,89,90,91,92,93"}}
16:29:11.758 RSL: STATUS5 = {"StatusNET":{"Hostname":"tasmota-09BB66-7014","IPAddress":"10.0.0.62","Gateway":"10.0.0.1","Subnetmask":"255.255.255.0","DNSServer1":"10.0.0.1","DNSServer2":"0.0.0.0","Mac":"C8:2B:96:09:BB:66","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
16:29:11.782 RSL: STATUS6 = {"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_09BB66","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
16:29:11.804 RSL: STATUS7 = {"StatusTIM":{"UTC":"2022-02-23T15:29:11","Local":"2022-02-23T16:29:11","StartDST":"2022-03-27T02:00:00","EndDST":"2022-10-30T03:00:00","Timezone":"+01:00","Sunrise":"07:43","Sunset":"18:23"}}
16:29:11.825 RSL: STATUS10 = {"StatusSNS":{"Time":"2022-02-23T16:29:11"}}
16:29:11.830 RSL: STATUS11 = {"StatusSTS":{"Time":"2022-02-23T16:29:11","Uptime":"0T00:05:33","UptimeSec":333,"Heap":18,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":20,"MqttCount":0,"Wifi":{"AP":1,"SSId":"","BSSId":"5A:83:C2:2D:CC:E7","Channel":11,"Mode":"11n","RSSI":78,"Signal":-61,"LinkCount":1,"Downtime":"0T00:00:03"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
1. Flash ESP8266 with tasmota-sensor
2. Wiring according to SHT30 Wiki article
3. Tasmota Settings for I2C in Generic Mode

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
Display Sensor Data Humidity, Temperature

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
![image](https://user-images.githubusercontent.com/1043739/155351519-bd3b15bf-206f-425d-909a-2dc6562d0d36.png)

![image](https://user-images.githubusercontent.com/1043739/155351770-df0d29b9-6361-4376-83c8-e715c399bce5.png)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
- Here is the sensor: [https://de.aliexpress.com/item/32964682265.html?gatewayAdapt=glo2deu&spm=a2g0o.9042311.0.0.7e784c4dbDD9oY](url)

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment, marked as answer]
What is the result of command 'i2cscan'?
It is preferable to disable unnecessary I2cdrivers with command 'i2cdriver<n> 0' and only keep the minimum number enabled

**Comment by rubenkr** [comment]
thank you very much @barbudor for helping me out!

18:09:42.799 RSL: RESULT = {"I2CScan":"No devices found"}

18:10:54.593 CMD: i2cdriver 0
18:10:54.602 RSL: RESULT = {"I2CDriver":"!1,2,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,28,29,30,31,32,34,35,36,40,41,42,43,44,45,46,48,49,50,51,52,53,55,56,58,59,60,61,62,63"}

I unscrewed the thermocoupler to see whats really inside. Is this really a SHT30 board?
![mhXOCta](https://user-images.githubusercontent.com/1043739/155374960-140048a2-7946-4c09-bdaf-48041cb8b1df.jpg)
![nMSk0ma](https://user-images.githubusercontent.com/1043739/155374973-6336d2ec-634c-4c52-86bd-3db1c4c0d7a6.jpg)

**Reply by barbudor** [reply]
May be....
But if it is, something is wrong as i2cscan is not finding any i2c device
Try out with another sensor if your setup is correct
